### DISCUSSION TITLE
Retrieval models supporting Arabic

### DISCUSSION THREAD

**Discussion by Fatima-Haouari** [castorini/pyserini#1187] [Q&A] [answered]
Hi,
Could you please help regarding the below:
Is there any models that supports retrieval for Arabic language other than BM25? how about neural retrieval models?

**Comment by lintool** [comment, marked as answer]
You might be interested in Mr.TyDi, which includes Arabic:
https://github.com/castorini/mr.tydi

**Reply by Fatima-Haouari** [reply]
Thanks a lot for sharing this. Kindly help regarding the below: 
I already used pyserini to create the sparse index but looking at the link you shared it seems I can create the dense index also using pyserini but bot really clear to me how to do so with for my own collection.

**Reply by Fatima-Haouari** [reply]
I was following this [link ](https://github.com/castorini/pyserini/blob/master/pyserini/resources/index-metadata/faiss.mrtydi-v1.1-arabic.20220207.5df364.README.md) 
I have these specific questions:
1) If my collection is in json files as required for sparse indexing. Can I use it for dense indexing or different files format is needed.
2) Do we have to specify --fields 
3) I believe --delimiter "\n\n" is not valid for json files
4) I am getting this error regarding the model  **raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://huggingface.co/api/models/models/mdpr-context-encoder**

**Reply by lintool** [reply]
Renamed model: https://huggingface.co/castorini/mdpr-passage-nq
